systemtest/src/test/resources/io/strimzi/systemtest/HelmChartST.testDeployKafkaClusterViaHelmChart.yaml

```diff
@@ -32,4 +32,6 @@ spec:
       type: ephemeral
     metrics:
       lowercaseOutputName: true
-  topicOperator: { }
\ No newline at end of file
+  entityOperator:
```

Comment:
Not for this PR, but are we really going to force people to use these 3 lines if they're happy with the defaults for both TO and UO? Can't `entityOperator: {}` mean the same as:
```
  entityOperator:
    topicOperator: {}
    userOperator: {}
```

Reply:
I think we discussed this with Paolo. But we would need a different mechanism to disable the operators.